Sources/NIO/ByteBuffer-int.swift

```diff
@@ -13,7 +13,8 @@
 //===----------------------------------------------------------------------===//
 
 extension ByteBuffer {
-    private func toEndianness<T: FixedWidthInteger> (value: T, endianness: Endianness) -> T {
+    @_inlineable @_versioned
```

Comment:
Is both right here?

Reply:
Yes, needs to be versioned so it can be used from inlineable but it can also be inlineable. Just gives the compiler more options.